.github/CODEOWNERS

```diff
@@ -1 +1,3 @@
 /codebuild @dougch
+
+/tests/cbmc @feliperodri @danielsn @lrstewart @camshaft @tautschnig @remi-delmas-3000
```

Comment:
camshaft and I probably shouldn't be on this list. We're not really a replacement for feliperodri or danielsn :)